src/cpp/src/image_generation/models/autoencoder_kl.cpp

```diff
@@ -223,11 +223,14 @@ AutoencoderKL& AutoencoderKL::compile(const std::string& device, const ov::AnyMa
     return *this;
 }
 
-ov::Tensor AutoencoderKL::decode(ov::Tensor latent) {
+ov::Tensor AutoencoderKL::decode(ov::Tensor latent, float& infer_duration) {
     OPENVINO_ASSERT(m_decoder_request, "VAE decoder model must be compiled first. Cannot infer non-compiled model");
 
     m_decoder_request.set_input_tensor(latent);
+    const auto infer_start = std::chrono::steady_clock::now();
     m_decoder_request.infer();
+    infer_duration = ov::genai::PerfMetrics::get_microsec(std::chrono::steady_clock::now() - infer_start);
```

Comment:
can external code (within Image generation pipeline) measure time of `decode` function? W/ such approach we don't need extra output arguments for this method.

The same approach for all other models within this `image_generation/models` folder.